doc/bitcoin-conf.md

```diff
@@ -62,3 +62,9 @@ Linux | `$HOME/.bitcoin/` | `/home/username/.bitcoin/bitcoin.conf`
 macOS | `$HOME/Library/Application Support/Bitcoin/` | `/Users/username/Library/Application Support/Bitcoin/bitcoin.conf`
 
 You can find an example bitcoin.conf file in [share/examples/bitcoin.conf](../share/examples/bitcoin.conf).
+
+### Command-line options and Security
+
+`-startupnotify`, `-shutdownnotify`, `-blocknotify`, `-walletnotify` and `-alertnotify` configuration parameters accept shell commands to be executed based on different events. This is useful for notifying the user of certain events, however it can be misused by malicious software or attackers using social engineering. Restrict scripts and commands in the shell/system used for bitcoind to avoid running potentially unsafe scripts.
```

Comment:
AFAIK #23395 is not yet merged, and hence `-shutdownnotify` has not yet been added to master. So you should remove that from the docs.

```suggestion
`-startupnotify`, `-blocknotify`, `-walletnotify`, and `-alertnotify` configuration parameters accept shell commands to be executed based on different events. These are useful for notifying the user of certain events; however, they can be misused by malicious software or social engineering attacks. Restrict scripts and commands in the shell/system used for bitcoind to avoid running potentially unsafe scripts.
```

Reply:
> AFAIK #23395 is not yet merged, and hence -shutdownnotify has not yet been added to master. So you should remove that from the docs.

This PR can be considered a part of #23395 so they should be merged together. It adds docs and I have suggested to add the link for this doc in release notes. There is no code.

Fix other errors in grammar in https://github.com/bitcoin/bitcoin/commit/78a364cf1a3ef89b8592c1fd81e794b535cfcd17